packages/space-opera/src/test/camera_settings/camera_settings_test.ts

```diff
@@ -22,19 +22,31 @@ import {CameraSettings, CameraTargetInput} from '../../components/camera_setting
 import {dispatchCameraTarget, dispatchInitialOrbit} from '../../components/camera_settings/reducer.js';
 import {Vector3D} from '../../components/camera_settings/types.js';
 import {dispatchAutoRotate, getConfig} from '../../components/config/reducer.js';
+import {getModelViewer} from '../../components/model_viewer_preview/reducer.js';
 import {reduxStore} from '../../space_opera_base.js';
 
 describe('camera constraints test', () => {
   let cameraSettings: CameraSettings;
 
   beforeEach(async () => {
+    expect(getModelViewer()).toBeUndefined();
+    document.body.innerHTML += `
+    <model-viewer-preview>
+      <model-viewer>
```

Comment:
Is this your test mock? Might be worth a comment.

Reply:
gotcha